dev/integration_tests/web/web/index_with_flutterjs_custom_sw_version.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE HTML>
+<!-- Copyright 2014 The Flutter Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file. -->
+<html>
+<head>
+  <meta charset="UTF-8">
+  <meta content="IE=Edge" http-equiv="X-UA-Compatible">
+
+  <title>Integration test. App load with flutter.js. Custom serviceWorkerVersion provided.</title>
+
+  <!-- iOS meta tags & icons -->
+  <meta name="apple-mobile-web-app-capable" content="yes">
+  <meta name="apple-mobile-web-app-status-bar-style" content="black">
+  <meta name="apple-mobile-web-app-title" content="Web Test">
+  <link rel="manifest" href="manifest.json">
+  <script>
+    // The value below is injected by flutter build, do not touch.
+    var serviceWorkerVersion = null;
```

Comment:
Please, remove this global from the DOM so it's not available to flutter.js, and it replicates more closely the repro case you prepared for the issue!

Reply:
Hmm, those are two different test cases, but this testing harness doesn't scale up very well, so OK, let's leave it like this.

Reply:
We'll clean-up the global once we introduce more complex variables to the index.html generation.

Reply:
I believe it is fine because the root cause is the same